Todo/AngularJS/Todo.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+
+<html ng-app="todoApp" ng-controller="controller" md-theme="IdoTheme">

```

Comment:
your ng-controller should be on a div inside the body, in case you want to add another one later